torchvision/models/vision_transformer.py

```diff
@@ -134,6 +143,7 @@ def __init__(
         num_classes: int = 1000,
         representation_size: Optional[int] = None,
         norm_layer: Callable[..., torch.nn.Module] = partial(nn.LayerNorm, eps=1e-6),
+        conv_stem_layers: Optional[List[ConvStemConfig]] = None,
```

Comment:
naming nit: `conv_stem_layers` looks like the layer instance while this is actually config, how about `conv_stem_config` or `conv_stem_params`?